queens-psyc-clinic/app
# FE User Cards Modal

### **User Cards Modal**
User cards modal component complete! Should be able to click any card and have a modal popping up displaying the information presented in the card. Buttons should also close the modal.

_Note: CardsModal.tsx was already merged to main so these are final edits._

### **Testing**
1. navigate to branch: `git checkout cards-modal`
2. `cd` to the `public` folder
3. run `npm install`
4. run `npm start`
5. ensure its working when you press any card and the modal information is matching:)
<br>

When you open the modal it should look like this:
![Screenshot 2024-03-13 at 9 09 06 PM](https://github.com/queens-psyc-clinic/app/assets/74020381/5d16879c-69e1-48c2-9a98-ba341df34e48)

For reference checkout issue: #84
